dashboard/app/views/levels/_dialogHelper.js

```diff
@@ -0,0 +1,57 @@
+/* global $ skipSound preTitle lastServerResponse Dialog */
+
+var dialogType = null;
+var adjustedScroll = false;
+
+$(function () {
+  if (preTitle != "") {
+    window.setTimeout(function () {
+      showDialog("pre");
+    }, 1000);
+  }
+});
+
+function dialogHidden() {
+  if (dialogType == "success" && lastServerResponse.nextRedirect) {
+    window.location.href = lastServerResponse.nextRedirect;
+  }
+
+  if (dialogType == "error") {
```

Comment:
Might as well make this === while in here. Look forward to being able to lint this stuff